lua/tardis/parts/legacy/legact_atomaccel.lua

```diff
@@ -6,6 +6,7 @@ PART.Name = "Legacy Atom Accelerator"
 PART.Model = "models/drmatt/tardis/atomaccel.mdl"
 PART.AutoSetup = true
 PART.Collision = true
+PART.BypassIsomorphic = false
```

Comment:
We do not need to include this false in all the parts, only true in the parts that require it